### DISCUSSION TITLE
Error parsing LastKnownRevisionDate

### DISCUSSION THREAD

**Discussion by chris01010100** [dani-garcia/vaultwarden#1775] [General]
Hello,

i use vaultwarden in unraid as docker.

If i create a new entry in vaultwarden with the Bitwarden iOS App (Ver. 2.10.0) then in the bitwarden logfile appear the following error warning:

`[2021-06-19 10:07:47.195][vaultwarden::api::core::ciphers][WARN] Error parsing LastKnownRevisionDate '0001-01-01T00:00:00': premature end of input`

If i create a new entry in the webvault or with the MacOS Bitwarden app (Ver. 1.26.5) this warning does not created.

What is happend here?

Thank You!
Chris

**Comment by jjlin** [comment]
#1779 will remove this warning, but in this case (creating a new entry), it's not a real problem.
